keyboards/nyhxis/nfr_70/rules.mk

```
@@ -0,0 +1,22 @@
+# MCU name
+MCU = atmega32u4
+
+#
+# LUFA specific
+#
+# Target architecture (see library "Board Types" documentation).
+ARCH = AVR8
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
```

Since it's pro micro compatible, ideally this should be set to that bootloader, by default
```suggestion
BOOTLOADER = caterina
```

---

The atmel-dfu bootloader would then have to be set by anyone who wishes to use the Elite-C?